Feature: Configuration file

Tony763 opened the pull request:
Address PR #12

Sorry I messed a rebase.

Commit:
Feature: Config file

lupus78 reviewed (changes requested):
can you rename paterns.txt to patterns.txt (with double t)

Tony763 commented:
Ouu, I will never ever learn.  not only tt but also this file is not mean to be in repository as every body can create it from scratch.
Removed, rebased.
